src/Polymesh.ts

```diff
@@ -387,11 +394,13 @@ export class Polymesh {
    *
    * @param args.owner - identity representation or Identity ID as stored in the blockchain
    *
-   * * @note reservations with unreadable characters in their tickers will be left out
+   * @note reservations with unreadable characters in their tickers will be left out
+   * @note supports pagination
    */
   public async getTickerReservations(args?: {
```

Comment:
Can't add pagination to this one because results are filtered on the client side